src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm/index.js

```diff
@@ -43,22 +43,19 @@ const StyledButton = styled(Button)`
 
 const DEFAULT_TEXT_FIELDS = 15;
 
-const DefaultTextsForm = ({ categoryUrl, state, defaultTexts, subCategories, onSubmitTexts, onOrderDefaultTexts }) => {
-  const form = useMemo(() => {
-    const fields = [...new Array(DEFAULT_TEXT_FIELDS).keys()].reduce(
-      (acc, key) => ({
-        ...acc,
-        [`item${key}`]: FormBuilder.group({
-          title: [''],
-          text: [''],
-        }),
-      }),
-      {}
-    );
-
-    return FormBuilder.group(fields);
-  }, []);
+const fields = [...new Array(DEFAULT_TEXT_FIELDS).keys()].reduce(
```

Comment:
Why do we need to do this global? Does useMemo make any sense then? How about using an object factory?

Reply:
It's not a global value; it's a var that is limited to the module's scope. `useMemo` Is needed to ensure that the `form` const doesn't change between renders.